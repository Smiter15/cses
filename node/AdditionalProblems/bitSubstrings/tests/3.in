000101001011010100
