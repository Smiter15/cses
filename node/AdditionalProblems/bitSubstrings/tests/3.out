16 34 31 23 20 21 14 12 0 0 0 0 0 0 0 0 0 0 0 
